modules/database/src/main/resources/database.properties

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###############################################################################
-database.url.host=${mysql.host}
-database.url.port=${mysql.port}
-database.url.dbname=${mysql.dbName}
-database.url.args=${mysql.Args}
+database.url.host:${mysql.host}
```

Comment:
I think we usually use `=` in the other property files. I would vote going with that, as it allows to also execute them to set variables in a shell.